app/screens/channel/index.tsx

```diff
@@ -102,22 +87,12 @@ const Channel = ({currentChannelId, currentTeamId, time}: ChannelProps) => {
                     testID='channel.post_list'
                     forceQueryAfterAppState={appState}
                 />
-                <View style={styles.sectionContainer}>
-                    <Text
-                        onPress={doLogout}
-                        style={styles.sectionTitle}
-                    >
-                        {`Loaded in: ${time || 0}ms. Logout from ${serverUrl}`}
-                    </Text>
-                </View>
-                <View style={styles.sectionContainer}>
-                    <Text
-                        onPress={goToAbout}
-                        style={styles.sectionTitle}
-                    >
-                        {'Go to About Screen'}
-                    </Text>
-                </View>
+                <PostDraft
+                    channelId={currentChannelId}
+                    screenId={''}
```

Comment:
Now that I think about it, probably was empty on V1 since it was on the root. Now that it is not on the root, probably it is better if we get the screenId of the channel screen. It is not currently available on the props, but I can probably raise it from somewhere (hopefully just adding the prop to the types here).

Reply:
After looking into it further, screenId is not being used, so I will remove it, at least for now.